addons/web/controllers/main.py

```diff
@@ -561,10 +561,11 @@ def get_qweb_templates_checksum(cls, addons=None, db=None, debug=False, bundle=N
     def get_qweb_templates(cls, addons=None, db=None, debug=False, bundle=None):
         return cls(addons, db, debug=debug)._get_qweb_templates(bundle)[0]
 
+
```

Comment:
useless diff

Reply:
This was to comply with E305:
expected 2 blank lines after class or function definition, found 1

(yes I have a linter know :) )

Should I remove it?